src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -318,10 +340,10 @@ private void EnqueueWork(bool highPriority, bool frozenPartialSemantics, Cancell
                         // Create a context to store pass the information along and collect the results.
                         context = new TaggerContext<TTag>(
                             oldState, frozenPartialSemantics, spansToTag, caretPosition, oldTagTrees);
-                        await ProduceTagsAsync(context, cancellationToken).ConfigureAwait(false);
+                        await ProduceTagsAsync(context, cancellationToken).ConfigureAwait(true);
 
                         return ComputeNewTagTrees(oldTagTrees, context);
-                    }, cancellationToken).ConfigureAwait(continueOnCapturedContext: calledFromJtfRun);
+                    }, cancellationToken).ConfigureAwait(true);
```

Comment:
as explained in the top comment in this method, we want 'CA(true)` for everything within this method.